kratos/includes/checks.h

```diff
@@ -42,6 +42,9 @@
 #define KRATOS_CHECK_STRING_CONTAIN_SUB_STRING(TheString, SubString) if (TheString.find(SubString) == std::string::npos ) \
 KRATOS_ERROR << "The string \"" << SubString << "\" was not found in the given string"
 
+#define KRATOS_STANDARD_CHECK_NEAR(a,b) if(std::abs(a - b) > Globals::Epsilon * (1 + std::abs(a) + std::abs(b))) KRATOS_ERROR << "Check failed because " << #a << " = " << a << \
```

Comment:
The check is useful for more than just tests. So it would make sense to split the check from the exception. You could put the check into a regular function, e.g., `bool IsNear(double a, double b);` in the Kratos namespace and call it here.

Reply:
Sure, I will do that. Thanks

Reply:
About the definition of Epsilon as a global, I just found it appealing to have the arbitrary constant centralised (though it needs not be fixed to the value I proposed). Rationalising my decision though, I can see the following advantages:

First, it makes it clear that a convention has been reached about a standard small value in Kratos, (at least partially) discouraging further ad-hoc definitions of the kind. 

Second, it becomes an accessible variable for everyone to use. For instance, it could be reused to create new macros etc.

Admittedly, these are not super strong reasons so let's see what people think.